### ISSUE TITLE
MultiColumn Website Documentation

### ISSUE THREAD

**Issue by ebarnard** [QuestPDF/QuestPDF#960]
I noticed the mention of MultiColumn layouts in the release notes for v2024.7.0. It would be great if there were some docs and example usage on the website, as there are for the other container types.

This might be completely the wrong place for this request so feel free to close.

**Comment by ebarnard** [comment]
Quickest issue turnaround I've ever seen! Those docs look great, and seem to cover everything.

One question about performance, is there much of a difference between `BalanceHeight(true)` and `BalanceHeight(false)`? It feels like `false` should be a lot quicker.

**Comment by MarcinZiabek** [comment]
You are right that the `BalanceHeight(true)` option involves complex layout calculations but only impacts the last page. Assuming your content is long enough, it has a diminishing impact on the performance.

**Comment by ebarnard** [comment]
Is that the expensive layout calculation from the warning box in the docs, or is it very expensive even with `BalanceHeight(false)`?

**Comment by MarcinZiabek** [comment]
Apologies for my late reply!

The calculation is computationally expensive in all cases, though its impact varies. If your content consists of simple text, it should perform well. However, for more complex structures, such as tables with thousands of cells, the overall processing cost may increase significantly.

Since there has been no recent activity, I’m closing this issue. If you have a similar request in the future, feel free to open a new GitHub issue. Thanks for your feedback! ❤️
